pkgs/applications/misc/inherd-quake/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, openssl
+, stdenv
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "quake";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "phodal";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256:1f7k68g18g3dpnrsmhgmz753bly1i3f4lmsljiyp9ap0c6w8ahgg";
```

Nit *(This is just my personal preference)*

```suggestion
    sha256 = "1f7k68g18g3dpnrsmhgmz753bly1i3f4lmsljiyp9ap0c6w8ahgg";
```